File: python/raindrops/raindrops.py
def convert(number):
    factors = {3: "Pling", 5: "Plang", 7: "Plong"}
    sounds = ""
    for divisor, sound in factors.items():
        if number % divisor == 0:
            sounds += sound
    return sounds if len(sounds) > 0 else str(number)

File: go/raindrops/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
	var ret string
	var sounds = []string{"Pling", "Plang", "Plong"}
	for i, divisor := range []int{3, 5, 7} {
		if number%divisor == 0 {
			ret += sounds[i]
		}
	}

	if len(ret) > 0 {
		return ret
	}

	return fmt.Sprintf("%v", number)
}
